src/lib/codes.js

```diff
@@ -79,6 +79,15 @@ const systems = {
   // Substance Admin Substitution
   "http://terminology.hl7.org/CodeSystem/v3-substanceAdminSubstitution": {
 	"url": "https://build.fhir.org/ig/HL7/UTG/CodeSystem-v3-substanceAdminSubstitution.json"
+  },
+
+  // Consent Policy and Scope Definitions
+"http://terminology.hl7.org/CodeSystem/consentpolicycodes": {
+  "url": "https://terminology.hl7.org/5.2.0/CodeSystem-consentpolicycodes.json"
+  },
+
+"http://terminology.hl7.org/CodeSystem/consentscope": {
+  "url": "https://terminology.hl7.org/5.2.0/CodeSystem-consentscope.json"
   }
```

Comment:
Aha this is my bad --- it's almost good EXCEPT the server at terminology.hl7.org doesn't return CORS headers correctly, so the requests fail. build.fhir.org seems to host all the same stuff with CORS set up correctly, so the two "url" values should be:

* https://build.fhir.org/ig/HL7/UTG//CodeSystem-consentpolicycodes.json
* https://build.fhir.org/ig/HL7/UTG//CodeSystem-consentscope.json

Reply:
no worries - so close! haha thank you